registrations/management/commands/revalidate_registrations.py

```diff
@@ -0,0 +1,102 @@
+from os import environ
+
+from django.core.management.base import BaseCommand, CommandError
+
+from registrations.models import Registration
+from registrations.tasks import (
+    add_personally_identifiable_fields, validate_subscribe)
+
+from seed_services_client import StageBasedMessagingApiClient
+
+from ._utils import validate_and_return_url
+
+
+class Command(BaseCommand):
+    help = ("Validates all invalid Registrations that failed with the given "
+            "error. This should also lead to the creation of a  Subscription "
+            "in the SMB service")
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--blind', action='store_false', default=True,
+            dest='check_subscription',
+            help=('Do not check with the stage based messaging API whether'
+                  'or not a subscription for the identity already exists.'
+                  'NOT RECOMMENDED AT ALL'))
```

Comment:
Fair point. I took this from one of the other management commands but I don't think we'd ever want to run this without checking first.